hack/test-go.sh

```diff
@@ -7,4 +7,5 @@ if [[ -n ${ARTIFACT_DIR:-} ]]; then
   JUNIT_ARG="--junitfile=$ARTIFACT_DIR/junit.xml"
 fi
 
-gotestsum $JUNIT_ARG -- ./... -race ${TESTFLAGS:-}
+set -o xtrace
+gotestsum $JUNIT_ARG -- ${PACKAGES:-"./..."} -race ${TESTFLAGS:-}
```

Comment:
We don't need the packages anymore now that w use buildflags, right?

Reply:
We do, because `go test ./... -tags e2e` will run the e2e and the rest, too

Reply:
(I think we could unset GOARCH or something to get around that? Unclear where the logical AND for the tags is coming from)